lang/cachyos-kernel-manager_tr.ts

```diff
@@ -0,0 +1,272 @@
+    <message>
+        <source>Running tick rate</source>
+        <translation>Çalışılan tik oranı</translation>
+    </message>
+    <message>
+        <source>Select tickless</source>
+        <translation>Tiksiz seçim yap</translation>
+    </message>
+    <message>
+        <source>Select preempt</source>
+        <translation>preempt belirle</translation>
+    </message>
+    <message>
+        <source>Enable multigenerational LRU</source>
+        <translation>Multigenerational LRU etkinleştirilsin</translation>
+    </message>
+    <message>
+        <source>Enable per-VMA locking</source>
+        <translation>per-VMA locking etkinleştirilsin</translation>
+    </message>
+    <message>
+        <source>Enable DAMON</source>
+        <translation>DAMON etkinleştirilsin</translation>
+    </message>
+    <message>
+        <source>Enable Linux Random Number Generator</source>
+        <translation>Linux Rastgele Sayı Üreteci etkinleştirilsin</translation>
+    </message>
+    <message>
+        <source>CPU compiler optimizations</source>
+        <translation>İşlemci derleme iyileştirmeleri</translation>
+    </message>
+    <message>
+        <source>Apply automatic CPU Optimization</source>
+        <translation>Otomatik işlemci iyileştirmeleri uygulansın</translation>
+    </message>
+    <message>
+        <source>Enable LTO</source>
+        <translation>LTO etkinleştirilsin</translation>
+    </message>
+    <message>
+        <source>Build the ZFS module</source>
+        <translation>ZFS modülünü yapılandır</translation>
+    </message>
+    <message>
+        <source>Build the BCACHEFS module</source>
+        <translation>BCACHEFS modülünü yapılandır</translation>
+    </message>
+    <message>
+        <source>Transparent Hugepages</source>
+        <translation>Saydam HugePage'ler</translation>
+    </message>
+    <message>
+        <source>Cancel</source>
+        <translation>Vazgeç</translation>
+    </message>
+    <message>
+        <source>Execute</source>
+        <translation>Gerçekleştir</translation>
+    </message>
+    <message>
+        <source>Custom package name</source>
+        <translation>Özel paket ismi</translation>
+    </message>
+    <message>
+        <source>$pkgbase</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>Build the NVIDIA module</source>
+        <translation>NVIDIA modülünü yapılandır</translation>
+    </message>
+</context>
+<context>
+    <name>ConfPatchesPage</name>
+    <message>
+        <source>Add remote patch</source>
+        <translation>Uzaktan yama ekle</translation>
+    </message>
+    <message>
+        <source>Add local patch</source>
+        <translation>Yerelden yama ekle</translation>
+    </message>
+</context>
+<context>
+    <name>ConfWindow</name>
+    <message>
+        <source>CachyOS Kernel Manager Configure</source>
+        <translation>CachyOS Çekirdek Yöneticisi Biçimleme</translation>
```

Comment:
"Yapılandır" instead of "Biçimleme"

Reply:
I also pondered on whether I should use "ayarla" or "yapılandır". An amenable proposition.